UI/src/view/linear_asset/speedLimitForm.js

```diff
@@ -94,8 +94,8 @@
         $('ul[class=information-content]').append('' +
           '   <li><a id="work-list-link-errors" class="wrong-speed-limits operator-user" href="#work-list/speedLimitErrors">Laatuvirheet Lista</a></li>' +
```

Comment:
check if List should start with lowercase too -> Laatuvirheet **Lista** -> Laatuvirheet **lista**